00-HTML-CSS-basics/The Basics of HTML/Questions.txt

```diff
@@ -1,5 +1,11 @@
+General:
+
+I accidentally pushed a commint to the master branch.
+Can I remove it somehow with rebase?
+
+
 HTML:
 
 1-5)
 Should I upload a resource folder containing the image, video and sound files?
```

Comment:
It's the same, for this case is not really necessary. In the future we will need you to upload the files.